yaqd_horiba/_horiba_ihr.py

```diff
@@ -25,6 +25,27 @@ class HoribaIHR320(HoribaMono):
     _kind = "horiba-ihr320"
     _ID_PRODUCT = IHR320_ID_PRODUCT
 
+    def __init__(self, *args, **kwargs):
+        startup_state = dict(self._state)
+        __super__().__init__(*args, **kwargs)
+
+        async def try_forever():
+            while True:
+                await self._not_busy_sig.wait()
+                for i in range(2):
+                    if startup_state["mirrors_dest"][i] != self._get_mirror(i):
+                        self._state["mirrors_dest"][i] = startup_state["mirrors_dest"][i]
+                        await self._reset_position()
+                        continue
```

Comment:
Unnecessary continue (it will continue the for loop not the whole loop, so it does nothing)

Reply:
You'll need to do some Boolean flagging, I think or putting else clauses on the for loops

Similarly, the break later will always run and thus the while loop will never loop

Reply:
Great catch, that's what I get for not turning on my brain :confounded: